### PR TITLE
feat(grpc-proxy): Refactor fern-bot as grpc-proxy server

### PR DESCRIPTION
**Pull request by dsinghvi**
Surge to refactor grpc proxy as a fargate server

### CODE REVIEW

File: packages/ui/fern-dashboard/src/lib/utils.ts
Comment (armandobelardo) [inline packages/ui/fern-dashboard/src/lib/utils.ts:1]: this seems unrelated, could you add it back

**Comment by dsinghvi** [comment]
was running into more issues, finally got the dumb lambda working
